src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -30,7 +30,7 @@ public class HomeWorkspaceModel : WorkspaceModel
         private readonly DynamoScheduler scheduler;
         private PulseMaker pulseMaker;
         private readonly bool verboseLogging;
-        private bool graphExecuted;
+        public bool graphExecuted;
```

Comment:
if you need to check this in a test, make it internal and use internalVisibleTo - there's likely other ways to do this though, like checking the node values.

Reply:
Changed this back to private. Not using this boolean anymore